css/desktop.css

```diff
@@ -111,53 +116,66 @@ header {
   .wrapper {
     .link-wrapper {
       display: grid;
-      grid-template-columns: repeat(4, 1fr);
-
-      row-gap: 40px;
-      column-gap: 20px;
-
-      .link-container {
+      grid-template-columns: repeat(4, 270px);
+      grid-template-rows: repeat(2, 320px);
+      row-gap: 50px;
+      column-gap: 30px;
+      justify-content: center;
+
+      .link-container-1, 
```

Comment:
Make common css. Class name can be duplicatated.
`.link-container { background-repeat: no-repeat; background-size: cover; position: relative; }`